### DISCUSSION TITLE
v3.0.3-beta

### DISCUSSION THREAD

**Discussion by mrghozzi** [mrghozzi/myads#32] [Announcements]
**Improvement**: multi-language
**Fix**: Some errors in templates
**Add**: Number of views for text ads

<hr /><em>This discussion was created from the release <a href='https://github.com/mrghozzi/myads/releases/tag/v3.0.3'>v3.0.3-beta</a>.</em>

**Comment by absouini** [comment]
i've translated it to french You can proceeed with [Autosurf et echange de banniere](https://surfer.42web.io/producer/francais_language_par_absouini)

**Reply by mrghozzi** [reply]
A new update has been added to the French language
[v3.0.4](https://github.com/mrghozzi/myads/releases)
